server/app.js

```diff
@@ -20,27 +18,16 @@ mongoose
 
 app.use(express.static(__dirname + '/public/resetpassword')); //connect with CSS and clear JS
 
-
-
-
-
-
-
 app.get('/passwordreset', (req, res) => {
   const fileName = path.join(__dirname, 'public/resetpassword/resetpassword.html');
   // res.sendFile(__dirname + '/public/resetpasswort/resetpasswort.html');
   res.sendFile(fileName);
 });
 
+// const reset = require('./routes/email/emailitems');
+// const token = reset.reset.token;
 
-
-
-const reset = require('./routes/email/emailitems');
-const token = reset.reset.token;
-
-
-app.get('/passwordcreate:' + token, (req, res) => {
-  console.log(token);
+app.get('/passwordcreate:resetToken', (req, res) => {
```

Comment:
```suggestion
app.get('/passwordcreate/:resetToken', (req, res) => {
```

Ten slash dużo ułatwi potem na froncie